src/olympia/devhub/urls.py

```diff
@@ -15,10 +15,8 @@
 submit_patterns = patterns(
     '',
     url('^$', lambda r, addon_id: redirect('devhub.submit.6', addon_id)),
-    url('^3$', views.submit_describe, name='devhub.submit.3'),
-    url('^4$', views.submit_media, name='devhub.submit.4'),
-    url('^5$', views.submit_license, name='devhub.submit.5'),
-    url('^6$', views.submit_done, name='devhub.submit.6'),
+    url('^details$', views.submit_describe, name='devhub.submit.4'),
+    url('^finish$', views.submit_done, name='devhub.submit.5'),
```

Comment:
Another good reason to do that is that the redirect to `'devhub.submit.6'` above that block is broken...